pkg/updater/updater_test.go

```diff
@@ -328,6 +337,22 @@ func (fs fakeStater) Stat(ctx context.Context, path gcs.Path) (*storage.ObjectAt
 
 type fakeUploader map[gcs.Path]fakeUpload
 
+func (fuc fakeUploader) Copy(ctx context.Context, from, to gcs.Path) error {
```

Comment:
nit: I think I'd use `fu` for a fakeUploader, to differentiate it from a `fakeUploaderClient`'s function

Reply:
Done